_data-prepper/pipelines/configuration/sources/s3.md

```diff
@@ -225,6 +228,7 @@ Option | Required | Type | Description
 `interval` | Yes  | String | Indicates the minimum interval between each scan. The next scan in the interval will start after the interval duration from the last scan ends and when all the objects from the previous scan are processed. Supports ISO_8601 notation strings, such as `PT20.345S` or `PT15M`, and notation strings for seconds (`60s`) and milliseconds (`1600ms`). 
 `count` | No | Integer | Specifies how many times a bucket will be scanned. Defaults to `Integer.MAX_VALUE`.
 
+
 ## Metrics
 
 The `s3` source includes the following metrics.
```

Comment:
```suggestion
The `s3` source includes the following metrics:
```

Reply:
I was told by @natebower to make this a period.